Moblin/HaishinKit/Media/AudioUnit.swift

```diff
@@ -133,26 +133,20 @@ final class AudioUnit: NSObject {
     }
 
     func addReplaceAudioPCMBufferInner(id: UUID, _ audioBuffer: AVAudioPCMBuffer) {
-        guard let mixer else {
-            return
-        }
         guard let replaceAudio = replaceAudios[id] else {
             return
         }
         let sampleBuffer = replaceAudio.createSampleBuffer(audioPCMBuffer: audioBuffer)
         guard let sampleBuffer, selectedReplaceAudioId != nil else {
             return
         }
-        let presentationTimeStamp = syncTimeToVideo(mixer: mixer, sampleBuffer: sampleBuffer)
-        guard mixer.useSampleBuffer(presentationTimeStamp, mediaType: AVMediaType.audio) else {
-            return
-        }
-        mixer.delegate?.mixer(
-            audioLevel: .infinity,
-            numberOfAudioChannels: sampleBuffer.formatDescription?.audioChannelLayout?.numberOfChannels ?? 0,
-            presentationTimestamp: presentationTimeStamp.seconds
+        var audioLevel: Float = .infinity
```

Comment:
This variable doesn't add any value imo. Just pass `.infinity` to the function below.